pkg/prompb/types.pb.go

```diff
@@ -2167,6 +2177,12 @@ func (m *TimeSeries) Unmarshal(dAtA []byte) error {
 	if iNdEx > l {
 		return io.ErrUnexpectedEOF
 	}
+	m.Labels = labelsPool[labelsPos:]
+	m.Samples = samplesPool[samplesPos:]
+	m.Exemplars = exemplarsPool[exemplarsPos:]
+	arrayPool.labelsPool = labelsPool
+	arrayPool.samplesPool = samplesPool
+	arrayPool.exemplarsPool = exemplarsPool
```

Comment:
nit: I think this is error prone. Let's just use arrayPool.labelsPool in the code directly rather than create intermediate variables

Reply:
Yeah I can see that. The reason I did it was to reduce verbosity of having `arrayPool.X` repeating. Anyways I don't feel strongly either way.